coursedata/adventures/de.yaml

```diff
@@ -320,71 +320,71 @@ adventures:
+        name: "Papagei"
+        description: "Erschaffe Dir deinen eigenen zahmen Papageien, der Dir alles nachplappert!"
         image: "story.png"
-        default_save_name: "Parrot"
+        default_save_name: "Papagei"
         levels:
             1:
                 story_text: |
-                    ## Parrot
-                    Create your own online pet parrot that will copy you!
+                    ## Papagei
+                    Erschaffe Dir deinen eigenen zahmen Papageien, der Dir alles nachplappert!
 
-                    ## Example Hedy code
+                    ## Hedy Beispiel-Code
                     ```
-                    print Im Hedy the parrot
-                    ask whats your name?
+                    print Ich bin Hedy der Papagei!
+                    ask Wie lautet Dein Name?
                     echo
                     echo
                     ```
-                start_code: "print Im Hedy the parrot"
+                start_code: "print Ich bin Hedy der Papagei!"
             2:
                 story_text: |
-                    ## Parrot
-                    Create your own online pet parrot that will copy you!
+                    ## Papagei
+                    Lass Deinen zahmen Papageien noch etwas echter klingen!
 
-                    ## Example Hedy code
+                    ## Hedy Beispiel-Code
                     ```
-                    print Im Hedy the parrot
-                    name is ask whats your name?
-                    print name
+                    print Ich bin Hedy der Papagei.
+                    Name is ask Wie lautet Dein Name?
+                    print Name
                     sleep
-                    print squawk
+                    print Aaaaaak
                     sleep
-                    print name
+                    print Name
                     ```
-                start_code: "print Im Hedy the parrot!"
+                start_code: "print Ich bin Hedy der Papagei!"
             3:
                 story_text: |
-                    ## Parrot
-                    Teach your parrot a new word with `add`.
+                    ## Papagei
+                    Bringe Deinem Papageien mit dem Befehl `add` neue Wörter bei.
 
-                    ## Example Hedy code
+                    ## Hedy Beispiel-Code
                     ```
-                    words is squawk, Hedy
-                    print Train your parrot!
-                    new_word is ask Which word do you want to teach them?
-                    add new_word to words
-                    print You: Say new_word , Hedy!
-                    print Hedy the Parrot: words at random
+                    Wörter is Aaaaaaak, Hedy
+                    print Bring Deinem Papageien etwas bei!
+                    Neues_Wort is ask Welches Wort willst Du ihm beibringen?
+                    add Neues_Wort to Wörter
+                    print 🧒 Sag Neues_Wort , Hedy!
```

Comment:
I like the usage of the emojis here but it is contradictory to the english version. fine for me, anyway.

Reply:
yes we just recently started to allow emoji so I guess they will seep into content slowly